src/Application/Dto/DistributionDto.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace LinkedDataSets\Application\Dto;
+final class DistributionDto
+{
+    private string $format;
+    private string $fileName;
+    private int $id;
+    public function __construct($format, $fileName, $id)
```

Comment:
I'd say, add type definition to the parameters of your construction. That ensures the correct data is loaded and fails early when it is not.